src/orders/components/OrderCustomerAddressesEditDialog/styles.ts

```diff
@@ -2,6 +2,14 @@ import { makeStyles } from "@saleor/macaw-ui";
 
 export const useStyles = makeStyles(
   {
+    scrollableContent: {
+      maxHeight: `calc(100vh - 250px)`,
```

Comment:
Where does `250px` come from? If it's used somewhere else (I suspect it is) maybe it would be better to make a const and use it here?

Reply:
It's not. Originally I thought about using theme.spacing but in this place 1 spacing is 0.8 rem, so to achieve desired result value needs to be something like `theme.spacing(26)`.

250px is arbitrary number leading to proper, good-looking margins on the top and bottom of the modal without making the css behaviour unexpected.

Reply:
Arbitrarily chosen value, it'll be removed in near future (this sprint), so I wouldn't overthink it